src/views/pages/HomeView/LandingPage.js

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import ArrowForwardOutlinedIcon from '@material-ui/icons/ArrowForwardOutlined';
+import {
+  Box,
+  Grid,
+  Typography,
+  makeStyles,
+  Button,
+  AppBar
+} from '@material-ui/core';
+import TopBar from 'src/layouts/MainLayout/TopBar';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    backgroundColor: '#FFF',
+    height: '90vh',
+    color: '#000',
+
+    overflow: 'hidden',
+    [theme.breakpoints.down('md')]: {
+      paddingTop: 0,
+      paddingBottom: 0
+    }
+  },
+  extraPadding: {
+    [theme.breakpoints.down('sm')]: {
+      padding: '0 30px'
+    },
+    textAlign: 'justify'
+  },
+  image: {
+    perspectiveOrigin: 'left center',
+    transformStyle: 'preserve-3d',
+    perspective: 1500,
+    width: '60vh',
+    display: 'flex',
+    '& > img': {
+      maxWidth: '100%',
+
+      backfaceVisibility: 'hidden'
+      // boxShadow: theme.shadows[16]
+      // transform: 'rotateY(-35deg) rotateX(15deg)'
+    },
+    [theme.breakpoints.down('sm')]: {
+      maxWidth: '40vh'
+    }
+  },
+  hide: {
+    display: 'none'
+  },
+  Button: {
+    textTransform: 'capitalize'
+    // padding :
+  },
+  container: {
+    paddingTop: '10vh',
+    margin: '0 5px',
+    [theme.breakpoints.down('sm')]: {
+      margin: '0',
+      paddingTop: '2vh',
```

Comment:
Try setting it to `6vh`, seems to fix the issue with tablet viewport here.

Reply:
By _it_ I mean `paddingTop`.

Reply:
Yes am also facing this issue , but this is only showing on chrome., on firefox its not happening